src/include/sof/audio/audio_stream.h

```diff
@@ -30,6 +30,34 @@
  *  @{
  */
 
+/**
+ * set of parameters describing audio stream
+ * this structure is shared between audio_stream.h and sink/source interface
+ * TODO: compressed formats
+ */
+struct sof_audio_stream_params {
+	enum sof_ipc_frame frame_fmt;	/**< Sample data format */
+	enum sof_ipc_frame valid_sample_fmt;
+
+	uint32_t rate;		/**< Number of data frames per second [Hz] */
+	uint16_t channels;	/**< Number of samples in each frame */
+
+	/** alignment limit of stream copy, this value indicates how many
+	 * integer frames which can meet both byte align and frame align
+	 * requirement. it should be set in component prepare or param functions
```

Comment:
this comment is very very unclear. What does 'byte align' mean? What does 'frame align' mean? 

I know the text is the same as before but I still can't figure out what the intent of this flag is.

Consider rewording or clarifying this field.

Reply:
This is NOT my comment, I just moved frame_align from buffer.h to audio_stream.h
I don't want to "fix" audio_stream in this PR, even if in comments

Reply:
@andrula-song can you please take the AR to fix those comments? 
they are confusing and make little sense.
Adding @singalsu as reviewer :-)

Reply:
here is the link https://github.com/thesofproject/sof/pull/7833